_configs/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "branches": [
+    "master",
+    "beta",
+    "release",
+    "esr115"
+  ],
+  "head": "master",
+  "paths": [
+    "browser/branding/official/locales",
+    "browser/extensions/formautofill/locales",
+    "browser/extensions/report-site-issue/locales",
+    "browser/locales",
+    "devtools/client/locales",
+    "devtools/shared/locales",
+    "devtools/startup/locales",
+    "dom/locales",
+    "mobile/android/branding/official/locales",
```

Comment:
This shouldn't be here? Noticed while trying out a script to automate the generation of this list.

Source: https://searchfox.org/mozilla-central/source/mobile/android/locales/l10n.toml

Reply:
I think it ended up here because I gathered the original list by looking for all the `locales/en-US` directories.

And now I'm starting to wonder what the right fix here is, because from a quick glance at the build configs it looks to me an awful lot like we _are_ using these strings on mobile. [Bug 1827759](https://bugzilla.mozilla.org/show_bug.cgi?id=1827759) might be related to this.